### PR TITLE
New version: Mixxx.Mixxx version 2.5.0

### LINKED ISSUE #202771: [Update Request]: Mixxx.Mixxx 2.5.0
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Mixxx.Mixxx

### Package Version

2.5.0

### Please describe the changes you would like to see

https://mixxx.org/news/2024-12-24-mixxx-2_5-released/

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #202771

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/202773)

### CODE REVIEW

File: manifests/m/Mixxx/Mixxx/2.5.0/Mixxx.Mixxx.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+InstallerSwitches:
+  InstallLocation: INSTALL_ROOT="<INSTALLPATH>"
+UpgradeBehavior: install
+ReleaseDate: 2024-12-21
+AppsAndFeaturesEntries:
+- ProductCode: '{660C0DCB-681F-4843-B210-293523BB8BD7}'
+  UpgradeCode: '{921DC99C-4DCF-478D-B950-50685CB9E6BE}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.mixxx.org/snapshots/2.5/mixxx-2.5-beta-152-g70d7999549-win64.msi
```
Comment (SpecterShell) [inline manifests/m/Mixxx/Mixxx/2.5.0/Mixxx.Mixxx.installer.yaml:17]: This is a beta version @Dragon1573

Comment (Dragon1573) [reply manifests/m/Mixxx/Mixxx/2.5.0/Mixxx.Mixxx.installer.yaml:17]: But [it says](https://mixxx.org/download/#stable) itself a stable version ... 😕

image

Comment (Dragon1573) [reply manifests/m/Mixxx/Mixxx/2.5.0/Mixxx.Mixxx.installer.yaml:17]: > [!WARNING]
> 
> Due to signing issues, we can't provide a signed `2.5.0` release now. In the meantime please use `2.5-beta-152` which uses already the final source code of _Mixxx_ `2.5.0`.
>
> _Originally posted at [Mixxx - Download the Best Free DJ Mixing Software App](https://mixxx.org/download/#stable)_

image

Comment (SpecterShell) [reply manifests/m/Mixxx/Mixxx/2.5.0/Mixxx.Mixxx.installer.yaml:17]: OK...I was wondering why my bot didn't work. 🤔

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**